src/components/layout/Main/Main.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react';
+import { fetchBlogs } from '../../../services/blogs.js';
+import BlogCard from '../../BlogCard/BlogCard.js';
+
+export default function Main() {
+  const [blogs, setBlogs] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const resp = await fetchBlogs();
+      setBlogs(resp);
+    };
+    fetchData();
+  }, []);
```

Comment:
this is fine and a normal pattern especially for such a small app but the rubric wanted you to move the state / useEffect logic into a separate custom hook (so you'd have a directory called hooks and a file called either blogs.js or useBlogs.js, do this logic in there, have that return the state variable)